src/map/npc.cpp

```diff
@@ -1986,6 +1986,7 @@ static int npc_selllist_sub(struct map_session_data* sd, int n, unsigned short*
 	int key_refine = 0;
 	int key_attribute = 0;
 	int key_identify = 0;
+	int key_uniqueid = 0;
```

Comment:
is it good to add bound info of item too ?

Reply:
this is a good idea , but it's better to have it in another PR as this PR does a specific thing and it might not accepted , however the item bound information onsell might be accepted later on it's on PR, ofc i would leave that for the developers to decide